src/Form/Field/ContentTypeFieldChoiceList.php

```diff
@@ -52,39 +63,52 @@ public function getStructuredValues()
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
+     *
+     * @return array<mixed>
      */
-    public function getOriginalKeys()
+    public function getOriginalKeys(): array
     {
         return $this->choices;
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
```

Comment:
remove {@inheritDoc} ?

Reply:
Referering to: Symfony\Component\Form\ChoiceList\ChoiceListInterface methods